set-web/src/main/java/eu/europeana/set/web/search/UserSetQueryBuilder.java

```diff
@@ -65,6 +63,16 @@ private UserSetQuery buildSearchQuery(Map<String, String> searchCriteria, String
 	    }
 	}
 
+	if (searchCriteria.containsKey(WebUserSetFields.SET_ID)) {
+		String setId = searchCriteria.get(WebUserSetFields.SET_ID);
+		if (setId != null && ! UserSetUtils.isSetIdNumeric(setId)) {
+			throw new ParamValidationException(I18nConstants.INVALID_PARAM_VALUE, I18nConstants.INVALID_PARAM_VALUE,
+					new String[] { "invalid value for search field " + WebUserSetFields.SET_ID,
```

Comment:
"invalid value for search field ", the information from the messages.en is sufficient

Reply:
done